prefect_dbt/cli/configs/base.py

```diff
@@ -0,0 +1,98 @@
+"""Module containing models for base configs"""
+
+from typing import Any, Dict, Optional
+
+from prefect.blocks.core import Block
+from pydantic import Field
+
+
+class DbtConfigs(Block):
```

Comment:
This should subclass `abc.ABC` so that it doesn't show up in the UI.

Reply:
This still needs to subclass `abc.ABC` because we don't want it to be registered and show in the UI